docs/use-cases/ai_workflow_automation/real-time-trading-forecasts.mdx

```diff
@@ -138,7 +138,7 @@ join_learn_process = true;
 INSERT INTO btcusdt_slack_app.messages (channel_id, text)
 VALUES("slack-channel-id", "Here are the BTCUSDT forecasts for the next 10 minutes:");
 INSERT INTO btcusdt_slack_app.messages (channel_id, text)
-SELECT "slack-channel-id" AS channel_id, concat('timestamp: ', cast(to_timestamp(cast(m.open_time as bigint)) as string), ' -> open price: ', m.open_price) AS text
+SELECT "slack-channel-id" AS channel_id, concat('timestamp: ', m.open_time, ' -> open price: ', m.open_price) AS text
```

Comment:
Same as above. Let's undo, unless we change the engine in this example to one of the Nixtla models (line 43 when creating a model).